financial-connections/src/main/java/com/stripe/android/financialconnections/features/linkaccountpicker/LinkAccountPickerViewModel.kt

```diff
@@ -284,8 +285,10 @@ internal class LinkAccountPickerViewModel @AssistedInject constructor(
         val drawerOnSelection = accounts.firstOrNull { it.first.id == partnerAccount.id }?.second?.drawerOnSelection
 
         if (drawerOnSelection != null) {
-            // TODO@carlosmuvi handle drawer display.
-            logger.debug("Drawer on selection: $drawerOnSelection")
+            presentNoticeSheet(
```

Comment:
We could rename this to `presentSheet`.

I also imagine that we can get rid of the subclasses and just have a single `presentSheet(SheetContent)` method, by using the generic screen data class for the existing `Legal` and `DataAccess` subclasses. What do you think?

Not something that needs to happen in this pull request. I’d be happy to do it as a follow-up!

Reply:
Agreed! Doing the renaming for now.

Reply:
Renamed it as part of https://github.com/stripe/stripe-android/commit/fc614e6b9a094515745be42cf3e32b60bae9dd64